src/Blocks/BlockDoor.cpp

```diff
@@ -61,13 +61,18 @@ void cBlockDoorHandler::OnUse(cChunkInterface & a_ChunkInterface, cWorldInterfac
 		case E_BLOCK_DARK_OAK_DOOR:
 		case E_BLOCK_JUNGLE_DOOR:
 		case E_BLOCK_SPRUCE_DOOR:
-		case E_BLOCK_IRON_DOOR:
 		case E_BLOCK_OAK_DOOR:
 		{
 			ChangeDoor(a_ChunkInterface, a_BlockX, a_BlockY, a_BlockZ);
 			a_Player->GetWorld()->BroadcastSoundParticleEffect(1003, a_BlockX, a_BlockY, a_BlockZ, 0, a_Player->GetClientHandle());
 			break;
 		}
+		// Prevent iron door from opening on player click
+		case E_BLOCK_IRON_DOOR:
```

Comment:
I don't think handling here is required as the client knows not to open an iron door.